anyway/language_constants.py

```diff
@@ -0,0 +1,34 @@
+class Hebrew(object):
+    ROAD = "כביש"
+    INTERCITY = "בינעירוני"
+
+    def __init__(self, *args, **kwargs):
+        super(Hebrew, self).__init__(*args, **kwargs)
```

Comment:
This constructor can be left out. Also, why instantiate this class at all? Even further, use a class at all? A static class is essentially a module.